### PR TITLE
documentation: clarify combined cache

### PR DESCRIPTION
**Pull request by sluongng**
In Bazel, a Combined Cache is in effect when both Local Disk Cache (--disk_cache=) and Remote Cache (--remote_cache=) are set. This concept, however, only exists in Bazel source code and not defined anywhere in Bazel's documentation.

Let's clarify Combined Cache concept in the doc to explain to users what it is.

### CODE REVIEW

**Review by coeuvre — APPROVED** [review]

**Comment by sluongng** [comment]
@sgowroji does this PR require +1 approval before it could be merged?

**Comment by sgowroji** [comment]
Adding "Awaiting-PR-merge" is fine to merge. @coeuvre Should we import this PR, as it is approved.

**Comment by coeuvre** [comment]
Yes, please!

**Pull request closed without merging**
